test/functional/mining_prioritisetransaction.py

```diff
@@ -120,7 +120,7 @@ def run_test(self):
         tx_id = self.nodes[0].decoderawtransaction(tx_hex)["txid"]
 
         # This will raise an exception due to min relay fee not being met
-        assert_raises_rpc_error(-26, "min relay fee not met (code 66)", self.nodes[0].sendrawtransaction, tx_hex)
+        assert_raises_rpc_error(-26, "min relay fee not met, 0 < 134 (code 66)", self.nodes[0].sendrawtransaction, tx_hex)
```

Comment:
Please don't hardcode `134`, as it will change all the time. Either calc the size or use a regex or something

https://travis-ci.org/bitcoin/bitcoin/jobs/371168622#L6011

Reply:
See 55f89da1a5048cf275c37bceef664e7c04831a35

Reply:
Ok, thanks, will look into this!